src/test/DeltaNeutralLpV3.t.sol

```diff
@@ -182,11 +190,29 @@ contract DeltaNeutralV3EthTest is DeltaNeutralV3Test {
         vm.createSelectFork("ethereum", 16_394_906);
     }
 
-    function _usdc() internal pure override returns (address) {
+    function _asset() internal pure override returns (address) {
         return 0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48;
     }
 
     function _deployStrategy() internal override {
         strategy = SslpV3.deployEth(vault);
     }
 }
+
+contract DeltaNeutralV3EthWethTest is DeltaNeutralV3Test {
+    function _selectFork() internal override {
+        vm.createSelectFork("ethereum", 16_394_906);
+    }
+
+    function _asset() internal pure override returns (address) {}
+
+    function _setAsset() internal virtual override {}
```

Comment:
`_setAsset` does nothing now, so the asset is just the [fake token](https://github.com/AffineLabs/contracts/blob/4c00d3f672e748c4eb39083e4c794feb000e55b8/src/test/Deploy.sol#L34)  deployed along with the L2Vault.

Reply:
No, the vault is not `L2Vault` here. It is a `Vault` deployed by the `EthVaults.deployEthWeth()`  function. See `_deployVault` override function.

Reply:
And this vault has real WETH token as asset. So no need to override. Also, FYI, overriding this new `Vault` asset fails. Something to look into if we have time. Can't find slot for `asset()`.

```
Running 1 test for src/test/DeltaNeutralLpV3.t.sol:DeltaNeutralV3EthWethTest
[FAIL. Reason: Setup failed: stdStorage find(StdStorage): Slot(s) not found.] setUp() (gas: 0)
Test result: FAILED. 0 passed; 1 failed; finished in 3.02s
```